app/controllers/errors_controller.rb

```diff
@@ -255,6 +255,13 @@ def create_pr
     project_scope = @current_project || @project
     @issue = (project_scope ? project_scope.issues : Issue).find(params[:id])
 
+    # Check quota and show warning
+    account = current_user&.account
```

Comment:
Since we use "current_user.account" a lot, it might make sense to put it in a method
```
def set_project
    @account = current_user&.account
  end
```

To avoid creating the "account" variable too often, think about it :)